DOCS/man/options.rst

```diff
@@ -2533,6 +2533,22 @@ Subtitles
     ``complex`` is the default. If libass hasn't been compiled against HarfBuzz,
     libass silently reverts to ``simple``.
 
+``--sub-ass-prune-delay=<-1|seconds>``
```

Comment:
This needs interface changes entry.

Reply:
Done but we should remove this line https://github.com/mpv-player/mpv/blob/master/DOCS/interface-changes.rst?plain=1#L20-L21

Reply:
I think we list each new option for a long time.